src/main/java/com/spotinst/sdkjava/model/api/azure/elastiGroup/V3/ApiElastigroupDetachedVmsAzure.java

```diff
@@ -0,0 +1,61 @@
+package com.spotinst.sdkjava.model.api.azure.elastiGroup.V3;
+
+import com.fasterxml.jackson.annotation.JsonFilter;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.spotinst.sdkjava.client.rest.IPartialUpdateEntity;
+
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+@JsonFilter("PartialUpdateEntityFilter")
+public class ApiElastigroupDetachedVmsAzure implements IPartialUpdateEntity {
+
+    @JsonIgnore
+    private Set<String>                                 isSet;
+    private List<ApiDetachVmsResponseDetachedVmsAzure>  detachedVms;
+    @JsonProperty("newVms")
```

Comment:
Recheck?

Reply:
tried removing newVms, but response is not mapped if we remove